src/llmq/options.cpp

```diff
@@ -132,7 +132,7 @@ bool IsQuorumTypeEnabledInternal(Consensus::LLMQType llmqType, gsl::not_null<con
             return true;
         case Consensus::LLMQType::LLMQ_50_60:
             return !fDIP0024IsActive || !fHaveDIP0024Quorums ||
-                    Params().NetworkIDString() == CBaseChainParams::TESTNET;
+                    Params().NetworkIDString() == CBaseChainParams::TESTNET || Params().NetworkIDString() == CBaseChainParams::DEVNET;
```

Comment:
maybe even all test nets?
```suggestion
                    Params().IsTestChain();
```

Reply:
We don't want to enable this for Regtest